internal/testing/e2e/docker.go

```diff
@@ -17,6 +19,10 @@ type dockerImageContext struct{}
 
 var dockerImageContextKey = dockerImageContext{}
 
+const (
```

Comment:
For my own knowledge, do we use parens for single-line declarations going forward? I've seen some inconsistency but wasn't sure what our team convention is.

Reply:
not sure we really have a consistent convention yet, generally I prefer `const` blocks to make it easier to add constants to in the future

Reply:
I've only used parens for multiple-delcaration, I don't _think_ I've seen parens used for singular much at all.